cmd/ocm-support/delete/capabilities/cmd.go

```diff
@@ -0,0 +1,80 @@
+package capabilities
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+
+	"github.com/openshift-online/ocm-cli/pkg/ocm"
+	v1 "github.com/openshift-online/ocm-sdk-go/accountsmgmt/v1"
+	"github.com/openshift-online/ocm-support-cli/cmd/ocm-support/utils"
+	"github.com/openshift-online/ocm-support-cli/pkg/label"
+	"github.com/openshift-online/ocm-support-cli/pkg/request"
+)
+
+var args struct {
+	filter   string
+	noDryRun bool
+}
+
+func init() {
+	flags := CmdDeleteCapability.Flags()
+	flags.StringVar(
+		&args.filter,
+		"filter",
+		"",
+		"If passed, filters and deletes the matching capabilities.",
+	)
+	flags.BoolVar(
+		&args.noDryRun,
+		"no-dry-run",
+		false,
+		"If passed, it will execute the delete command call in instead of a dry run.",
```

Comment:
Consider using "If false" to match the pattern used on previously created boolean flags.